include/linker/ramfunc.ld

```diff
@@ -0,0 +1,20 @@
+SECTION_DATA_PROLOGUE(.ramfunc,,)
+{
+#ifdef CONFIG_MPU_REQUIRES_POWER_OF_TWO_ALIGNMENT
+	. = ALIGN( 1 << LOG2CEIL(_ramfunc_ram_end - _ramfunc_ram_start));
```

Comment:
@qianfan-Zhao below this line we need to align minimum with 32-byte, to avoid MPU issues with ARMv8-M, which requires non-overlapping regions, and 32-byte minimum MPU region